### DISCUSSION TITLE
August 2021 release (DeGoogle your Life)

### DISCUSSION THREAD

**Discussion by seanpm2001** [seanpm2001/Degoogle-your-life#14] [New release]
***

# August 2021 release (DeGoogle your Life)

### Changes

This is the first release of the DeGoogle Your Life project and it is coming a bit late. Since this is the first release, all changes made are just what is in the repository so far. This version was released on 2021 August 31st as an end of month update. It includes:

- [x] The main DeGoogle your life article, with its first 7 versions

- [x] GitHub sponsor support

- [x] Social preview enabled

- [x] Discussion enabled

- [x] Research reference directory

- [x] Roadmap directory for article changes

- [x] End goal center, a place for discussing the achievement of DeGoogling your life

- [x] Update plans directory for update plans

- [x] Project language file 1 (HTML5)

- [x] Support for X-Text files (`AUTHORS` `CREDITS` `COPYING` `INSTALL`)

- [x] Makefile support

- [x] Git file support (`.gitattributes` `.gitignore` `.gitmodules` `.editorconfig`)

- [x] Translations into several dozens of languages

### Release info

**Commit count:** `254`

**Commits in last release:** `NaN`

**Commit change:** `No previous releases, NaN`

**Status:** `Production ready, rolling updates`

**Release date:** `2021 Tuesday, August 31st at 2:47 pm)`

***

<hr /><em>This discussion was created from the release <a href='https://github.com/seanpm2001/Degoogle-your-life/releases/tag/2021.08.31'>August 2021 release (DeGoogle your Life)</a>.</em>

**Comment by NET6666STREEM** [comment]
Amazing Work got to help protect the little MAN hypothetically speaking. I have a bit more than a few Project's in the Bugging stages patiently looking for those fit for the job an advisor to be the best of we can make it only together well can make the world a better place. Can't do big changes individually for those like myself. To humble persons Stay humbling and Thank You! What little humanity is left still Prevailing in the end! #BLINDPETTE
